regression/kernel/kmalloc_invalid_allocation/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+test.c
+-I /home/kai/PGT/ESBMC_Project/esbmc/src/c2goto/library/  
```

Comment:
We should avoid this. 

@rafaelsamenezes : can you suggest an alternative here?

Reply:
Installing the new headers in the c2goto directory should be enough to be able to clear this line.